### ISSUE TITLE
README.md not found

### ISSUE THREAD

**Issue by daisukelab** [torchsynth/torchsynth#311]
Hi, I got an error when trying "Which way to synth1B1-312-6?".
```
Traceback (most recent call last):
  File "t.py", line 3, in <module>
    from torchsynth.synth import Voice
  File "/home/xx/anaconda3/lib/python3.8/site-packages/torchsynth/__init__.py", line 1, in <module>
    from torchsynth.__info__ import (  # noqa: F401
  File "/home/xx/anaconda3/lib/python3.8/site-packages/torchsynth/__info__.py", line 20, in <module>
    __long_docs__ = open(os.path.join(PATH_ROOT, "README.md"), "r", encoding="utf-8").read()
FileNotFoundError: [Errno 2] No such file or directory: '/home/xx/anaconda3/lib/python3.8/site-packages/torchsynth/../README.md'
```
Just avoided with a workaround to `touch /home/xx/anaconda3/lib/python3.8/site-packages/README.md` but maybe it would be better not to try to load that.
Thanks

**Comment by jorshi** [comment]
Hi, thanks for letting us know -- this is fixed up on main and will be released on pypi soon. 

Cheers,
Jordie

**Comment by jorshi** [comment]
We've updated the version on pypi and this is fixed in 0.9.2
